ibc-clients/ics07-tendermint/types/src/client_state.rs

```diff
@@ -394,7 +394,7 @@ pub(crate) mod serde_tests {
     fn serialization_roundtrip_no_proof() {
         let json_data = include_str!(concat!(
             env!("CARGO_MANIFEST_DIR"),
-            "/../../../ibc-testkit/tests/data/json/client_state.json"
+            "/../../../tests-integration/tests/data/json/client_state.json"
```

Comment:
can you please look if we can move these tests inside `tests-integration`?

Reply:
[668df68](https://github.com/cosmos/ibc-rs/pull/1233/commits/668df682241111a0ff853c6ceda0eb1d0d18b13e)